system-tests/lib/cre/environment/environment.go

```diff
@@ -337,8 +344,8 @@ func SetupTestEnvironment(
 	// Ugly glue hack ¯\_(ツ)_/¯
 	fullCldInput := &keystonetypes.FullCLDEnvironmentInput{
 		JdOutput:          jdOutput,
-		BlockchainOutput:  blockchainsOutput.BlockchainOutput,
-		SethClient:        blockchainsOutput.SethClient,
+		BlockchainOutput:  homeChainOutput.BlockchainOutput,
+		SethClient:        homeChainOutput.SethClient,
```

Comment:
shouldn't we rather have an indication (e.g. using `chainselector`) of which chain is home chain and initialise CLD environment for all chains? and to have a `map[uint64]*seth.Client` instead of a single Seth?